apps/forms/templates/widgets/plea_radio_field.html

```diff
@@ -5,90 +5,131 @@
 {% load debug %}
 
 {% with plea_data=plea.data|index:index %}
-
 {% get_current_language as LANGUAGE_CODE %}
 
-<div class="form-group-wide{% if form.guilty.errors %} with-error{% endif %}" id="section_guilty">
+<div class="form-group-wide{% if form.guilty.errors %} with-error{% endif %}"
+     id="section_guilty">
 
     <fieldset role="radiogroup" aria-labelledby="label_charge">
 
         <legend>
+
             <div class="header-line" id="label_charge">
-                <span class="label-text">{% spaceless %}
```

Comment:
Didn't seem to be doing anything useful. If we can avoid needing it that would be good.